tests/test_pipelines/test_loading.py

```diff
@@ -61,6 +63,47 @@ def test_load_img(self):
         np.testing.assert_equal(results['img_norm_cfg']['mean'],
                                 np.zeros(1, dtype=np.float32))
 
+    def test_load_multi_imgs(self):
+        results = dict(
+            img_prefix=self.data_prefix,
+            target_imgs_info=dict(filename='color.jpg'),
+            ref_imgs_info=dict(filename=['color.jpg', 'color.jpg']))
+        transform = LoadMultiImagesFromMultiFiles()
+        results = transform(copy.deepcopy(results))
+        assert results['target_filenames'] == [
+            osp.join(self.data_prefix, 'color.jpg')
+        ]
+        assert results['ref_filenames'] == [
+            osp.join(self.data_prefix, 'color.jpg'),
+            osp.join(self.data_prefix, 'color.jpg')
+        ]
+        assert results['target_ori_filenames'] == ['color.jpg']
+        assert results['ref_ori_filenames'] == ['color.jpg', 'color.jpg']
+        assert results['target_img_0'].shape == (288, 512, 3)
+        assert results['target_img_0'].dtype == np.uint8
+        assert results['ref_img_0'].shape == (288, 512, 3)
+        assert results['ref_img_0'].dtype == np.uint8
+        assert results['ref_img_1'].shape == (288, 512, 3)
+        assert results['ref_img_1'].dtype == np.uint8
+        assert results['img_shape'] == (288, 512, 3)
+        assert results['ori_shape'] == (288, 512, 3)
+        assert results['pad_shape'] == (288, 512, 3)
+        assert results['scale_factor'] == 1.0
+        assert repr(transform) == transform.__class__.__name__ + \
+            "(\n\tprefixs=['target', 'ref'],\n" + \
+            '\tto_float32=False,\n' + \
+            "\tcolor_type='color',\n" + \
+            "\tfile_client_args={'backend': 'disk'})\n"
+
+        # img_info_keys not exists
+        results = dict(
+            img_prefix=self.data_prefix,
+            support_img_info=dict(filename='color.jpg'),
+            reference_img_info=dict(filename=['color.jpg', 'color.jpg']))
+        transform = LoadMultiImagesFromMultiFiles()
+        with pytest.raises(KeyError):
+            transform(copy.deepcopy(results))
```

Comment:
Add a comment indicating what assertion you are testing.

Reply:
Done